### ISSUE TITLE
Unclear senteces in Healthcheck/Rules/RuleDescription.resx

### ISSUE THREAD

**Issue by ruppde** [vletoux/pingcastle#172]
The following sentences from Healthcheck/Rules/RuleDescription.resx are somehow strange, but I couldn't get my head around them to fix it:

```
Passwords stored in clear text or obfuscated can be retrieved. By reusing the user's identity, an attacker does not need to perform attack and it is difficult to detect it.

If the password is a secret which protects its derivatives, such as the fingerprint named hash, can be used as if it was the password itself.

Check if there is the privilege Access Credential Manager as a trusted caller user right has been explicitly granted.

In short, a connection to the computer account AZUREADSSOACC and the secret of this user account is used as a shared secret with AzureAD.

The recommended way to deal with this situation, is to make sure that no objects are using these attributes or to set the value com.sun.jndi.ldap.object.trustURLCodebase to false to ALL the Java code.
```

Not sure if "side channel attack" is the right word here?
```
Changing regularly secrets like passwords ensures that they are not used in side channel attacks.
```

**Comment by vletoux** [comment]
`Passwords stored in clear text or obfuscated can be retrieved. By reusing the user's identity, an attacker does not need to perform attack and it is difficult to detect it.
`
This is the category description of PasswordRetrieval
The idea is to provide a generic description of rules behind
![image](https://user-images.githubusercontent.com/10632326/229304172-1d73fd94-9771-4380-9caa-2cff19a1831b.png)


same for `If the password is a secret which protects its derivatives, such as the fingerprint named hash, can be used as if it was the password itself.`

`Check if there is the privilege Access Credential Manager as a trusted caller user right has been explicitly granted.` has the value `P_TrustedCredManAccessPrivilege_Title`. It is then the Title of the rule P-TrustedCredManAccessPrivilege.
The goal is to check for the privilege TrustedCredManAccess that shouldn't be used.

For `In short, a connection to the computer account AZUREADSSOACC and the secret of this user account is used as a shared secret with AzureAD.` the AZUREADSSOACC hash is like the krbtgt hash but for AzureAD.

`The recommended way to deal with this situation, is to make sure that no objects are using these attributes or to set the value com.sun.jndi.ldap.object.trustURLCodebase to false to ALL the Java code.` is related to the rule S-JavaSchema. You can publish code base in the active directory. So the solution is not to use AD to perform the same functionality

**Comment by vletoux** [comment]
The technical name of the password fingerprint is « hash »
